### ISSUE TITLE
Add the LanCommunicator class ( Start , Stop, Send ) - Networking Module - 111701017

### ISSUE THREAD

**Issue by sunny117** [convict-git/KoraKaagaz#50]
This class instance will be created by LanCommunicatorFactory by giving port number (on which we will be listening for messages) for the constructor.

In Start method, we will be starting the communication on the network

Three queues will be created : 
1. Send Queue ( which contains the message that needs to be send on the network )
2. Receive Queue for Processing ( which contains the messages that we received from the network for the processing module )
3. Receive Queue for Content ( which contains the messages that we received from the network for the content module )

  Four Threads will be created : 
1. Send Queue Listener ( which takes messages from the send queues and sends on the network )
2. Socket Listener ( which listens on a socket and sends the messages received from the network to the respective receive queues based on the identifier )
3. Receive Queue Listener for Processing ( which takes messages from the processing receive queue and sends it through the handler )
4. Receive Queue Listener for Content ( which takes messages from the content receive queue and sends it through its handler )

In Stop method,
Basically communication will be stopped in this method, all the threads and instances will be stopped and made null.

In Send method, 
Processing and Content module will be using this method with destination, message and identifier as arguments. A object will be created with this arguments and pushed into the queue and later will be sent through the network handled by the Send Queue listener.
